pkg/iter/entry_iterator.go

```diff
@@ -200,55 +206,85 @@ func (i *mergeEntryIterator) Next() bool {
 	// preserve their original order. We look at all the top entries in the
 	// heap with the same timestamp, and pop the ones whose common value
 	// occurs most often.
+Outer:
```

Comment:
Is there a reason why this label is title case, whereas the inner is lower case? I'm not sure what the best practice is, but I would keep it consistent.

Reply:
https://stackoverflow.com/questions/64039172/whats-the-standard-practice-for-naming-labels-in-go 🤷